### PR TITLE
Add: polite closing socket; Change: player ID from int to string;

### PR DESCRIPTION
**Pull request by VirrageS**

### CODE REVIEW

File: test_player.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def test_quit_command(self):
         valid_parameters = VALID_ARGS()[1]
 
         with player_context(valid_parameters, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL) as program:
-            time.sleep(PLAYER_BOOT_SECONDS) # lepiej poczekac, bo dziwne akcje odwala...
+            time.sleep(PLAYER_BOOT_SECONDS)
```
Comment (apaszke) [inline test_player.py]: Przerzucisz to proszę do `player_context`? Tak będzie prościej, bo i tak wszędzie to mamy.

Comment (VirrageS) [inline test_player.py]: Zrobione

Comment (VirrageS) [inline test_player.py]: Jednak działa bez tego `PLAYER_BOOT_SECONDS`. Być może player mi coś wtedy słabo działał jak robiłem te testy.

**Comment by mluszczyk** [comment]
Why -11? (I mean the return code)

**Comment by VirrageS** [comment]
Nwm... fixed

File: test_master.py [hunk 2]
```diff
@@ -78,9 +78,10 @@ def test_not_a_number_at_all(self):
 
 class TestCommands(unittest.TestCase):
     def assertOK(self, line):
-        self.assertTrue(line.startswith(b"OK "))
-        player_id = int(line[3:]) # make sure the rest is convertible to int
-        return player_id
+        splited = line.split()
```
Comment (mluszczyk) [inline test_master.py]: should be "split" (grammar) ;)

**Comment by mluszczyk** [comment]
Looks good for me, but maybe @apaszke can have a look. I can't check it against my master, because it has never worked.

File: test_master.py (hunk 2 above)
Comment (VirrageS) [inline test_master.py]: hah, sure ^^

**Comment by apaszke** [comment]
I'll take a look in an hour or so.

**Comment by apaszke** [comment]
Works for me 👍

**Pull request merged**